publish-framework.sh

```diff
@@ -0,0 +1 @@
+pod repo push bryjai-bryj-build-private-pods Build-V3-Plugin-Optimizers-ios.podspec --allow-warnings
```

Comment:
Could you please let me know why we are adding publish-framework.sh & validate-framework.sh? 

Do we need to add for all others plugins?

Reply:
Yes @piyush-bryj, we will need to add it to other plugins. Both files were in most of the plugins implemented by Jerome and they allow for other devs to validate and publish the pods into Bryj internal repo without having to know any of the commands